src/gitlab/configuration-model.ts

```diff
@@ -209,6 +209,20 @@ export interface Service {
   readonly entrypoint?: string[];
   /** Full name of the image that should be used. It should contain the Registry part if needed.*/
   readonly name: string;
+  /** The pull policy that the runner uses to fetch the Docker image */
+  readonly pullPolicy?: PullPolicy[];
+  /** Additional environment variables that are passed exclusively to the service.. */
+  readonly variables?: Record<string, number | string>;
```

Comment:
In jsii languages this doesn't work very well. Let's just use a string here
```suggestion
  readonly variables?: Record<string, string>;
```

Reply:
@mrgrain this was copied from the `IncludeRule` section, where this value was already defined. I'm mentioning, because if we have a potential issue here, we may have it there too

Reply:
Thanks @ctasada Acknowledged. The other definition will be wrong as well. Also confirmed that this indeed should be a string: https://docs.gitlab.com/ee/ci/yaml/#variablesvalue

Can you make the change for you PR and we can fix the other occurrence in a separate PR (would love if you did that, but not worries if not 😅 )

Reply:
@mrgrain Done. Fixed both definitions